### ISSUE TITLE
The header section contains a link back to previous page

### ISSUE THREAD

**Issue by r-ferrier** [geeksforsocialchange/teamwilder#306]
I'm just wondering how necessary this is? Browsers have a back button already for this behaviour & in order to implement we'll need to pass forward information about the previous page to the next model.

**Comment by undividual** [comment]
Autumn in agreement that users to tend to default to using back button — so probably not worth the effort. One of our user testers did say 'I should never have to use the back button' — but we don't think it's worth it for the effort! 

We did discuss whether it might be worth having 'go home' link for users who might be unfamiliar with the fact that clicking the site name on every page would take them home, but decided not worth it for now.

**Comment by undividual** [comment]
How hard would it be to implement a breadcrumb instead of this, as per attached wireframe? This was another suggestion that came up in our meeting and it was discussed on the MVP Miro.

Autumn is imagining that many guides would fall under particular categories (like the "Get connected" etc bits I made on the homepage) so the breadcrumb would include the contents pages for each of those categories as I have tried to indicate here.

I am aware that implementing a breadcrumb would likely need to be a new ticket if we do decide to do it, but I don't want to create this in case it's far too complex for this milestone. Let me know what you think.
![MVPWireframes_2](https://github.com/geeksforsocialchange/teamwilder/assets/90701712/f1d79f36-9468-4380-9dd1-274dac442b7a)
